docker-compose.yml

```diff
@@ -13,6 +13,7 @@ web:
   volumes:
     - .:/src
   environment:
+    DJANGO_LOG_LEVEL: 'INFO'
```

Comment:
I would put this in the .env file,  already set to `'INFO'`. If we put it there, then we cannot override from the .env file